setup.cfg

```diff
@@ -3,3 +3,11 @@
 # 3. If at all possible, it is good practice to do this. If you cannot, you
 # will need to generate wheels for each Python version that you support.
 universal=1
+
+[semantic_release]
+version_variable = iceprod/__init__.py:__version__
+upload_to_pypi = False
+# patch_without_tag = True
+commit_parser = semantic_release.history.tag_parser
+minor_tag = [minor]
+fix_tag = [fix]
```

Comment:
GitHub says "No newline at end of file."  Should I ask for that to be changed? :smile:

Reply:
ahh yes :sunglasses: